lib/utils/mailup.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * Implements MailUp REST API
+ */
+
+import * as request from "request";
+import * as winston from "winston";
+
+import { Either, isLeft, left, right } from "fp-ts/lib/Either";
+
+// request timeout in milliseconds
+const TIMEOUT = 10000;
```

Comment:
better name it `TIMEOUT_MS` then, always useful to keep info about metric together with var (wither with tagged type or else in var name)

Reply:
also I guess this is actually a `DEFAULT_REQUEST_TIMEOUT_MS`